lib/AST/Attr.cpp

```diff
@@ -468,6 +468,12 @@ bool DeclAttribute::printImpl(ASTPrinter &Printer, const PrintOptions &Options,
     }
     break;
   }
+
+  case DAK_PrivateImport: {
+    Printer.printAttrName("@_private(sourceFile: \"");
```

Comment:
Nitpick: The parens and labels shouldn't be part of `printAttrName` (that's what controls the highlighting). Take a look at DAK_Available.

Reply:
I see.